bundle/manifests/k8s-nim-operator.clusterserviceversion.yaml

```diff
@@ -3,7 +3,19 @@ kind: ClusterServiceVersion
 metadata:
   name: nim-operator-certified.v0.1.0
   namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
```

Comment:
For longer term, at one time we need to add arm64 support for grace / blackwell? I believe gpu operator already does that?

Reply:
yes, whenever NIMs are ready for arm64